pages/common/fish.md

```diff
@@ -23,3 +23,7 @@
 - Display version information and exit:
 
 `fish --version`
+
+- Set and export universal environment variables:
```

Comment:
What are universal environment variables ? Is this a commonly used option ? Our scope in tldr is to mainly keep the pages to only use the common options for a user to get started.

Reply:
Universal variables persist across restarts, similar to adding it to the .bashrc in bash. For most use cases for new users of fish shell and/or command line in general, I feel like this is the way users would want to set environmental variables. 

Does `Set and export environmental variables that persist across restarts` explain the intent better?

Reply:
That's much better, thanks !